### PR TITLE
Add metadata for 21.5.5 du models

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/IdCards__1__metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "IDCards",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",  
+  "retrainable": true,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v20.7.1",
```
Comment (vivekjhauipath) [inline metadata/IdCards__1__metadata.json:20]: @irinel-birsan-uipath this should atleast be 20.10.1.2, and maybe 21.4. Language version wasn't supported till 20.10.1

File: metadata/Passports__1__metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "Passports",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",  
+  "retrainable": true,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v20.7.1",
```
Comment (vivekjhauipath) [inline metadata/Passports__1__metadata.json:20]: same

**Review by vivekjhauipath — CHANGES_REQUESTED** [review]
Change min supported version for new models

File: metadata/IdCards__1__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/IdCards__1__metadata.json:20]: Yes, my bad, copy-paste from wrong place. Will do

File: metadata/Passports__1__metadata.json (hunk 2 above)
Comment (irinel-birsan-uipath) [reply metadata/Passports__1__metadata.json]: done

**Review by vivekjhauipath — APPROVED** [review]
LGTM

**Review by suvhrajit-uipath — APPROVED** [review]

**Pull request merged**